docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -2163,8 +2163,10 @@ Keywords related to plotting
     points in ERTs flow of execution where you can hook in a workflow,
     before the simulations start, :code:`PRE_SIMULATION`; after all
     the simulations have completed, :code:`POST_SIMULATION`; before the
-    update step, :code:`PRE_UPDATE` and after the update step,
-    :code:`POST_UPDATE`.  The :code:`POST_SIMULATION` hook is
+    update step, :code:`PRE_UPDATE`; :code:`POST_UPDATE`; after the update
```

Comment:
Should we guarantee the order of `PRE_UPDATE` and `PRE_FIRST_UPDATE`?

Reply:
Yes, that makes sense. Will look a bit into refactoring the different models I think, there has to be enough overlap to do that.

Reply:
Not the order, but the refactor :arrow_up:

Reply:
Thanks for the heads up! Will leave it as is then, just assert the order in a test.